### DISCUSSION TITLE
Strona z rekomandacją MS dotycząca odpowiedniego rozmiaru vmki

### DISCUSSION THREAD

**Discussion by vonbartas** [AzureCommunityPL/group#382] [Pomoc] [answered]
Cześć,
Widziałem kiedyś filmik gdzie gość pokazywał stronkę (stronka MS) na której jest kilka pytań dot. obciążenia, innych parametrów i na koniec dostaje rekomendację MS jaki rozmiar vmki jest odpowiedni. 
Ma może ktoś linka do tej stronki?

Pzdr

**Comment by JanuszNowak** [comment]
@vonbartas chodzi ci pewnie o Azure Advisor 
link do dokumentacji i filmiku 
https://youtu.be/cSNPoAb-TNc
https://docs.microsoft.com/pl-pl/azure/cost-management-billing/costs/tutorial-acm-opt-recommendations
https://docs.microsoft.com/pl-pl/azure/advisor/advisor-cost-recommendations

**Comment by vonbartas** [comment]
Dzięki @JanuszNowak ale to nie to :( 
Tamta wyglądała coś jak Azure Calc.

**Reply by JanuszNowak** [reply]
@vonbartas  sprawdź 3 minutę filmu
![2022-01-12 12_52_39-Optimizing cloud investments in Azure Cost Management - YouTube](https://user-images.githubusercontent.com/5168275/149135652-7b73da48-856f-48d4-9250-7c1a4099c3b0.png)

**Comment by JanuszNowak** [comment]
@vonbartas na koniec filmiku masz zbiór linków odnośnie kosztów itd moze tam będzie ?
![2022-01-12 12_54_51-Optimizing cloud investments in Azure Cost Management - YouTube](https://user-images.githubusercontent.com/5168275/149135908-419bfd7f-6325-477d-ad21-7f60162b3a09.png)

**Reply by vonbartas** [reply]
Nie było to nic związane z Azure Advisor. Zwykła stronka z odpowiedziami na pytania...

**Comment by Swiderek** [comment, marked as answer]
Może o to chodzi https://azure.microsoft.com/pl-pl/pricing/vm-selector/
:-)

**Reply by vonbartas** [reply]
Dziękuję bardzo @Swiderek 🍻
